code/frameworks/vue-webpack5/package.json

```diff
@@ -68,6 +66,8 @@
     "@babel/core": "*",
     "babel-loader": "^7.0.0 || ^8.0.0 || ^9.0.0",
     "css-loader": "*",
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.8.0 || ^17.0.0 || ^18.0.0",
```

Comment:
Well the react frameworks depend on `@storybook/react` which has a peerDependency on `react`.
But I guess you have a point when saying `@storybook/vue` doesn't have this.

BUT, the builder has a peerDependency on react, this is because of `@storybook/manager-api`.

---

We can investigate if we can remove those aliasses, and thus remove those peerDependencies in 7.x